### PR TITLE
Tooltip shows complete domain name, issue 594

### PR DESCRIPTION
**Pull request by nextLane**
PB dropdown is too short and narrow #594

Changes in UI, now entire domain-name is showed as a tooltip thus solving the first problem of "dropdown being too narrow so domain-names get needlessly truncated and it becomes impossible to determine what 1st-level domain it belongs to."

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -309,19 +309,15 @@ function _trim(str,max){
...
-  var statusMap = { 
-    block:        status_block,
-    cookieblock:  status_cookieblock,
-    noaction:     status_noaction
+function _badgerStatusTitle(action , origin){ 
+  
+  var statusMap = {
+    block: "Blocked ",
+    cookieblock: "Blocked cookies from ",
+    noaction: "Allowed "
```
Comment (SwartzCr) [inline src/popup.js]: have these strings be loaded from i18n like the previous strings were

**Comment by SwartzCr** [comment]
this is really close to being perfect, but there's one more thing I'd like you to do.
Notice that previously it had been getting the text for these actions from the i18n script - this is what allows privacy badger to be translated into other languages.
If you could add your text to the english text file _locales/en_US/messages.json much like the previous ones (badger_status_block, badger_status_cookieblock, and badger_status_noaction) were, with similar names
Additionally you should remove those strings from the messages.json file if they're no longer being used in the code.
Let me know if you have any questions about this, but once you do that this should be ready to merge!

**Comment by nextLane** [comment]
Any other changes to be done?

**Comment by SwartzCr** [comment]
looks good!
I'm not going to merge it right now, because this is a change that will break other language's support, and I want to make sure I have a plan with how to deal with that before putting it in master. But consider your work done!

**Comment by cooperq** [comment]
This is great! Thank you! I think that perhaps the best way to deal with this for other locals is to replace the changed strings with the english versions and let volunteers translate them again. Or we could try to translate them ourselves using google translate.

**Comment by cooperq** [comment]
Looks great! Thank you! :+1:

**Pull request merged**